scss/_variables.scss

```diff
@@ -1989,4 +1989,40 @@ $quantity-selector-icon-sm-remove-height: .125rem !default;
 $quantity-selector-input-max-width:       2.625rem !default;
 $quantity-selector-input-sm-max-width:    2.5rem !default;
 // scss-docs-end quantity-selector
+
+//// Orange footer
+// scss-docs-start orange-footer
+$footer-font-size-sm:                        $font-size-sm !default;
+$footer-line-height-sm:                      $line-height-sm !default;
+$footer-font-size-md:                        $font-size-base !default;
+$footer-line-height-md:                      $line-height-base !default;
+$footer-letter-spacing:                      $letter-spacing-base !default;
+$footer-accordion-line-height:               1.8125rem !default;
```

Comment:
For all those values in `rem` there's no link at all with other variables;`$spacer` or others?